distributionviewer/core/static/js/app/components/views/chart.js

```diff
@@ -132,11 +135,11 @@ export class Chart extends React.Component {
     }
 
     if (this.props.showOutliers) {
-      graphOptions.min_y = this.pointsMeta[0].y;
-      graphOptions.max_y = this.pointsMeta[pointsMetaLength - 1].y;
+      graphOptions.min_x = this.pointsMeta[0].x;
+      graphOptions.max_x = this.pointsMeta[pointsMetaLength - 1].x;
```

Comment:
The charts were accurate before, but because `min_y` was often higher than 0 and `max_y` was often lower than 100, those tick marks weren't always shown.

By instead changing `min_x` and `max_x` based on the value of `this.props.showOutliers`, we can say elsewhere that `min_y` should always be 0 and `max_y` should always be 100, guaranteeing that they show up.